packages/cli/lib/services/dryrun.service.ts

```diff
@@ -394,4 +388,133 @@ export class DryRunService {
             process.exit(1);
         }
     }
+
```

Comment:
moving runScript into `dryrun.service`. I don't really see the point of having it in a different file

Reply:
note: 
idealistically we should be able to have the script runner as a dedicated package and use it in the backend + in the cli

Reply:
agreed. At the end that is probably the only thing that is common between the two paths. Not sure it will stay this way when we implement sandboxing though

Reply:
cloudflare wrangled is doing that I think https://github.com/cloudflare/workers-sdk